libs/api/domains/vehicles/src/dto/getVehicleDetailInput.ts

```diff
@@ -0,0 +1,17 @@
+import { Field, InputType } from '@nestjs/graphql'
+import { IsString } from 'class-validator'
```

Comment:
Don't need class-validator for graphql inputs (unless they're complex, like email validation etc.)

Reply:
Ah I see - fixed ✅